languagetool-language-modules/sr/src/main/resources/org/languagetool/rules/sr/grammar-spelling.xml

```diff
@@ -0,0 +1,22 @@
+<?xml-stylesheet type="text/css" href="../../../../../../../../../languagetool-core/src/main/resources/org/languagetool/rules/rules.css"
+        title="Easy editing stylesheet" ?>
+<!-- English Grammar and Typo Rules for LanguageTool See tagset.txt for the
```

Comment:
This comment still refers to "English", it should be updated or removed.

Reply:
Fixed that, and other similar comments. Pending commit.